assets/queries/ansible/aws/iam_access_key_is_exposed/test/positive.yaml

```diff
@@ -0,0 +1,44 @@
+- name: Create two new IAM users with API keys
+  community.aws.iam:
+    iam_type: user
+    name: "{{ item }}"
+    state: present
+    password: "{{ temp_pass }}"
+    access_key_state: active
+  loop:
+    - jcleese
+    - mpython
+
+
+- name: Create two new IAM users with API keys
+  community.aws.iam:
+    iam_type: user
+    name: "{{ item }}"
+    state: present
+    password: "{{ temp_pass }}"
+    access_key_state: active
+  loop:
+    - root
+    - mpython
```

Comment:
This case shouldn't be expected as well, on the line 19?

Reply:
yes it is creating two users. One is root and the other is mpython. So the mpython should not have a access_key_state:active.